specifications/api/swagger.yaml

```diff
@@ -281,6 +288,25 @@ x-walletAccountPubkey: &walletAccountPubkey
   maxLength: 128
   example: 1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1
 
+x-cosigner: &cosigner
+  type: string
+  pattern: "^(cosigner#)[0-9]*$"
+
+x-cosigners: &cosigners
+  type: object
+  properties:
+   cosigner#0: *walletAccountPubkey
+   cosigner#1: *walletAccountPubkey
+   cosigner#2: *walletAccountPubkey
+   cosigner#3: *walletAccountPubkey
+   cosigner#4: *walletAccountPubkey
+   cosigner#5: *walletAccountPubkey
+   cosigner#6: *walletAccountPubkey
+   cosigner#7: *walletAccountPubkey
+   cosigner#8: *walletAccountPubkey
+   cosigner#9: *walletAccountPubkey
+   cosigner#10: *walletAccountPubkey
```

Comment:
```yaml
  type: object
  additionalProperties: *walletAccountPubKey
  propertyName: "^(cosigner#)[0-9]*$"
```

Reply:
cannot accomplish more than : 
```
x-cosigners: &cosigners                                                                                                                                                                                        
  description: Map of cosigners and their account public key. Use key as in &cosigner, eg. "cosigner#"                                                                                                         
  type: object                                                                                                                                                                                                 
  additionalProperties: *walletAccountPubkey
```
it seems we need swagger 3.1 rather than 3.0 -> https://stackoverflow.com/questions/46552863/how-to-write-openapi-3-swagger-specification-for-property-name-in-map-object

Reply:
I don't think [openapi-spec-validator](https://github.com/p1c2u/openapi-spec-validator) supports 3.1 yet, so perhaps comment out the propertyName line and add a note. I did this before for the tx metadata.